sagemaker-experiments/sagemaker_job_tracking/pytorch_script_mode.ipynb

```diff
@@ -0,0 +1,548 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    #TODO: Delete later
</pre><p>Yea, need delete line 2-32 and upgrade sagemaker directly </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3703/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>